miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -190,12 +260,189 @@ def volume(self) -> str:
     def voice_package(self) -> str:
         return self.data["voice_package"]
 
+    @property
+    def timezone(self) -> str:
+        return self.data["timezone"]
+
+    @property
+    def cleaning_time(self) -> str:
+        return self.data["cleaning_time"]
+
+    @property
+    def cleaning_area(self) -> str:
+        return self.data["cleaning_area"]
+
+    @property
+    def first_clean_time(self) -> str:
+        return self.data["first_clean_time"]
+
+    @property
+    def total_clean_time(self) -> str:
+        return self.data["total_clean_time"]
+
+    @property
+    def total_clean_times(self) -> str:
+        return self.data["total_clean_times"]
+
+    @property
+    def total_clean_area(self) -> str:
+        return self.data["total_clean_area"]
+
+
+class Dreame1CVacuumStatus(DreameVacuumStatusBase):
```

Comment:
Please use a single status container. For the not-supported information, you can mark them as optional:
```
@property
def water_flow(self) -> Optional[WaterFlow]:
    if "water_flow" not in self.data:
        return None
    ...
```

This way we will have a single API for all Dreame devices (much like we have for Viomi and Roborock) that'll make it easier to consolidate the APIs among vendors in the future.